opencga-app/app/scripts/docker/opencga-init/override-yaml.py

```diff
@@ -59,6 +62,13 @@
         storage_config["cellbase"]["database"]["hosts"].clear()
     storage_config["cellbase"]["database"]["hosts"].insert(i, cellbase_host)
 
+# Inject Hadoop ssh configuration
+if args.hadoop_ssh_host and args.hadoop_ssh_user and args.hadoop_ssh_password:
+    storage_config["storageEngines"][1]["variant"]["options"] = {}
```

Comment:
Doing this you will drop any other options available in the original configuration file

Reply:
Yeh sorry this was left over from when I thought we were targetted `storage_config["storageEngines"][1]["options"]` which I wasn't sure if always existed. Will fix now, good spot!